nixos/tests/tor.nix

```diff
@@ -21,8 +21,11 @@ rec {
     };
 
   testScript = ''
-    $client->waitForUnit("tor.service");
-    $client->waitForOpenPort(9051);
-    $client->succeed("echo GETINFO version | nc 127.0.0.1 9051") =~ /514 Authentication required./ or die;
+    client.wait_for_unit("tor.service")
+    client.wait_for_open_port(9051)
+    if not "514 Authentication required." in client.succeed(
```

Comment:
Can't we use assert here?
```py
assert "514 Authentication required." in client.succeed("echo GETINFO version | nc 127.0.0.1 9051")
```

and it should raise an `AssertionError`?

Reply:
Yeah, `foo or die` idiom should be rewritten to `assert foo`